tests/inet_socket/test

```diff
@@ -325,24 +324,49 @@ if ($test_ipsec) {
 "runcon -t test_inet_bad_client_t -- $basedir/client dgram 127.0.0.1 65535 2>&1";
     ok( $result >> 8 eq 8 );
 
-    # Verify that unauthorized client cannot communicate with the server.
-    $result = system
-"runcon -t test_inet_bad_client_t -- $basedir/client dgram ::1 65535 2>&1";
-    ok( $result >> 8 eq 8 );
-
     # Kill the server.
     server_end($pid);
 
+    if ($test_ipv6) {
+
+        # Start the IPv6 stream server.
+        $pid = server_start( "-t test_inet_server_t", "-6 stream 65535" );
+
+        # Verify that authorized client can communicate with the server.
+        $result = system
+          "runcon -t test_inet_client_t $basedir/client stream ::1 65535";
+        ok( $result eq 0 );
+
+        # Verify that unauthorized client cannot communicate with the server.
+        $result = system
+"runcon -t test_inet_bad_client_t -- $basedir/client stream ::1 65535 2>&1";
+        ok( $result >> 8 eq 5 );
+
+        # Kill the server.
+        server_end($pid);
+
+        # Start the IPv6 dgram server.
+        $pid = server_start( "-t test_inet_server_t", "-6 dgram 65535" );
+
+        # Verify that unauthorized client cannot communicate with the server.
+        $result = system
+"runcon -t test_inet_bad_client_t -- $basedir/client dgram ::1 65535 2>&1";
+        ok( $result >> 8 eq 8 );
+
+        # Kill the server.
+        server_end($pid);
+
 # Start the dgram server for IPSEC test using IPv6 but do not request peer context.
```

Comment:
Perhaps this is just the GH interface playing tricks when displaying the patch, but it looks like the comment above isn't indented properly; if this is true, is this a mistake or intentional?  If intentional, why?

Reply:
`perltidy` won't let me align it since it's longer than 80 characters... Shall I rather split it into two lines?

Reply:
Yes, that is normally what one when do in such a situation.

Reply:
OK, done.